### PR TITLE
adding bms to sidebar

### PR DESCRIPTION
**Pull request by supunkamburugamuve**

### CODE REVIEW

File: website/sidebars.json [hunk 1]
```diff
@@ -1,8 +1,10 @@
 {
   "docs": {
     "Image Processing": ["img_proc"],
+    "Biomolecular Simulations": ["bms", "bms-cpptraj", "bms-mdanalysis", "bms-pmda", "bms-radical-cybertools"],
```
Comment (orbeckst) [inline website/sidebars.json]: This looks good.

File: docs/publications.md [hunk 2]
```diff
@@ -0,0 +1,360 @@
...
+
+## Rutgers University
+
+
+1. Hadoop on HPC: Integrating Hadoop and Pilot-based Dynamic Resource
+   Management Andre Luckow, Ioannis Paraskevakos, George
+   Chantzialexiou, Shantenu Jha Workshop on High-Performance Big Data
+   Computing, 2016 [`link <http://arxiv.org/abs/1602.00345>`_] [`bib
+   <https://www.github.com/saga-project/radical.wp/raw/master/publications/bib/hadoop-on-hpc.bib>`_]
+   : hadoop-on-hpc `http://arxiv.org/abs/1602.00345
```
Comment (orbeckst) [inline docs/publications.md]: These are restructured text links but the format is markdown – needs to be changed.

File: docs/publications.md [hunk 3]
```diff
@@ -0,0 +1,360 @@
...
+   <http://arxiv.org/abs/1602.00345>`_
+2. Pilot-Abstraction: A Valid Abstraction for Data-Intensive Application
+   on HPC, Hadoop and Cloud Infrastructures?, Andre Luckow,
+   Pradeep Mantha, Shantenu Jha pdf version
+   `here <http://arxiv.org/pdf/1501.05041v1.pdf>`__
```
Comment (orbeckst) [inline docs/publications.md]: Link names that just say "here" are semantically awful. Make the link name appear as an identifier such as 
```markdown
... [arXiv: 1501.05041v1](http://arxiv.org/pdf/1501.05041v1.pdf) ...
```

File: docs/publications.md [hunk 4]
```diff
@@ -0,0 +1,360 @@
...
+3. A Tale of Two Data-Intensive Paradigms: Applications, Abstractions,
+   and Architectures, Shantenu Jha, Judy Qiu, Andre Luckow,
+   Pradeep Mantha, Geoffrey C. Fox 2014 IEEE International Congress on
+   Big Data (BigData Congress), 2014 pdf version
+   `here <http://arxiv.org/abs/1403.1528>`__
```
Comment (orbeckst) [inline docs/publications.md]: avoid "here" links – see https://www.w3.org/QA/Tips/noClickHere

File: docs/publications.md [hunk 5]
```diff
@@ -0,0 +1,360 @@
...
+4. I. Paraskevakos, A. Luckow, M. Khoshlessan, G. Chantzialexiou,
+   T. E. Cheatham, O. Beckstein, G. Fox, and S. Jha. Task-parallel
+   analysis of molecular dynamics trajectories. In ICPP 2018: 47th
+   International Conference on Parallel Processing, August 13–16,
+   2018, Eugene, OR, USA, page Article No. 49, New York, NY, USA,
+   August 13–16 2018. Association for Computing Machinery, ACM. DOI:
+   [10.1145/3225058.3225128](https://doi.org/10.1145/3225058.3225128)
```
Comment (orbeckst) [inline docs/publications.md]: example of a semantically sensible link

**Review by orbeckst — CHANGES_REQUESTED** [review]
Thanks for adding the links to the BMS pages. This looks ok.

The PR also contains publication updates. I added some from ASU.

However, the link format appears to be restructured text while the file format is Markdown so most of the links need to be fixed. Furthermore, many links are [click here](https://www.w3.org/QA/Tips/noClickHere) type links that really should be changed to something that makes more sense semantically. I have examples in the PR that show what I mean.

**Comment by orbeckst** [comment]
@supunkamburugamuve let me know when you had time to work on the PR and I will review again.

File: docs/publications.md (hunk 5 above)
Comment (supunkamburugamuve) [reply docs/publications.md]: Added these changes

**Review by orbeckst — APPROVED** [review]

**Comment by orbeckst** [comment]
@supunkamburugamuve thanks – please merge when you think you're ready

**Pull request merged**